compair/tests/api/test_answers.py

```diff
@@ -239,6 +252,22 @@ def test_get_all_answers(self):
             self.assertEqual(20, rv.json['per_page'])
             self.assertEqual(len(self.fixtures.answers), rv.json['total'])
 
+            # test sorting by score
+            # for instructors, all comparable answers should be included, regardless whether they are compared or not
+            rv = self.client.get(self.base_url + '?orderBy=score')
+            self.assert200(rv)
+            result = rv.json['objects']
+            # test the result is paged and sorted
+            expected = sorted(
+                [answer for answer in self.fixtures.answers if answer.comparable],
+                key=lambda ans: (ans.score.score if ans.score else float('-inf'), ans.created),
+                reverse=True)
+            self.assertEqual([a.uuid for a in expected[:20]], [a['id'] for a in result])
+            self.assertEqual(1, rv.json['page'])
+            self.assertEqual(2, rv.json['pages'])
+            self.assertEqual(20, rv.json['per_page'])
+            self.assertEqual(len(expected), rv.json['total'])
+
```

Comment:
It would be good to fetch the second page here as well since that is where all the uncompressed answers should be